src/main/scala/com/gu/support/catalog/ProductRatePlan.scala

```diff
@@ -0,0 +1,20 @@
+package com.gu.support.catalog
+
+import com.gu.i18n.Currency.GBP
+
+case class ProductRatePlan(
+  id: String,
+  status: Status,
+  name: Option[String],
+  productRatePlanCharges: List[ProductRatePlanCharge]
+) {
+  def price: Double = {
+    productRatePlanCharges
+      .filter(_.endDateCondition.contains("Subscription_End"))
+      .map(
+        _.pricing
+          .filter(_.currency == GBP)
```

Comment:
Don't we ever need the price in other currencies?

Reply:
I still need to do some work on the catalog stuff, I should probably have left it out of this PR to be honest but since it's there I'm going to leave it in and sort it out later